vignettes/getting_started.Rmd

```diff
@@ -0,0 +1,194 @@
+---
+title: "Setting up checklist"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Setting up checklist}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Starting an R package from scratch
+
+### Prepare online setup
+
+1. Go to https://github.com and login.
+  Create a new public repository in the INBO organisation.
+  Use the package name as repository name.
```

Comment:
Are there naming conventions for package/repository names?  In this case you might want to refer to it here (preferably by using a link to keep the workflow clean).

Reply:
I'm not aware of any documentation on that.

Reply:
Do we have internal agreements on this (lowercase, capitals, only R in uppercase,...), or can we just pick any name?  If we can just pick any name, you can just ignore this remark.